app/utils/newsFeed/redis/mappings.ts

```diff
@@ -132,6 +137,7 @@ export const mapCollaborationQuestionToRedisNewsFeedEntry = (
   followedBy: RedisUser[],
 ): RedisNewsFeedEntry => {
   const item = mapToRedisCollaborationQuestion(question, reactions, followedBy);
+  updateImageUrls(item);
```

Comment:
The type `RedisCollaborationQuestion` has a field `authors` which is an array of users, which has to be mapped